src/main/java/org/datadog/jmxfetch/reporter/Reporter.java

```diff
@@ -70,7 +70,7 @@ public void sendMetrics(LinkedList<HashMap<String, Object>> metrics, String inst
             String[] tags = Arrays.asList((String[]) metric.get("tags")).toArray(new String[0]);
 
             // StatsD doesn't support rate metrics so we need to have our own aggregator to compute rates
-            if (!"gauge".equals(metricType)) {
+            if ("counter".equals(metricType)) {
```

Comment:
Indeed ! I will reverse the condition to explicitly catch "gauge" and "histogram".
I guess that for the next major release, we should clarify the condition to only accept known types.